packages/react/src/components/Footer/LegalNav.js

```diff
@@ -20,8 +22,8 @@ const { prefix } = settings;
 const renderTrusteItem = () => {
   return (
     <li
-      className={`${prefix}--legal-nav__list-item`}
-      data-autoid="dds-privacy-cp"
+      className={`${cPrefix}--legal-nav__list-item`}
+      data-autoid={`${prefix}--dds-privacy-cp`}
```

Comment:
`dds--dds-privacy-cp` is this what we want it to be? My guess the original `dds-` here was intended to be the updated prefix.